src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,11 +2,15 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        logger.info("Login method was called. Params : login={}",login);
```

Comment:
The log message is good but it could be more informative. For example, you can include information about the success or failure of the operation. It's not a critical issue, but it's a good practice to make log messages as informative as possible. For example: logger.info("Method login was called. Params: login={}, success={}", login, success); where success is a boolean that indicates whether the login operation was successful or not.